Gemfile

```diff
@@ -51,13 +53,15 @@ gem 'devise_token_auth', '>= 1.2.0', git: "https://github.com/lynndylanhurley/de
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'debug', platforms: %i[mri mingw x64_mingw]
 end
 
 group :development do
+  gem 'annotate', '~> 3.2'
```

Comment:
You're also adding this gem. That might be part of your PR description as well 🙂

Reply:
Thanks, I added to the descriptions.